src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,9 +80,24 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 loading => loading.Value || repositories.UnfilteredCount > 0 && !LoadingFailed)
                 .ToProperty(this, x => x.FilterTextIsEnabled);
 
-            this.WhenAny(x => x.IsLoading, x => x.LoadingFailed,
-                (loading, failed) => !loading.Value && !failed.Value && repositories.UnfilteredCount == 0)
-                .Subscribe(x => NoRepositoriesFound = x);
+            this.WhenAny(
+                x => x.repositories.UnfilteredCount,
```

Comment:
Calling the constructor and then Listen is literally the same thing as calling the constructor with an observable - it calls the other constructor and then it calls Listen. If there are duplicated items, then it's because the Comparer is not returning true when two different instances of an Item compare to the same thing. This likely means that the test is incorrectly implementing the necessary comparer to return the item equality.